tox.ini

```diff
@@ -0,0 +1,15 @@
+[tox]
+envlist = py27
+
+[base]
+deps =
+	coverage
+
+[testenv:py27]
+deps=
+	{[base]deps}
+	-r{toxinidir}/tests/requirements.txt
```

Comment:
I would prefer to get those dependencies added directly here, so that we don't need that extra requirements file anymore.